GUI/GUIListBox.cpp

```diff
@@ -188,33 +188,23 @@ void GUIListBox::ReceiveSignal(GUIPanel *Source, int Code, int Data)
 {
     if (Source->GetPanelID() == GetPanelID())
     {
-        // MouseMove
-        if (Code == GUIListPanel::MouseMove)
```

Comment:
I assume it's fine to replace the ifs with else-ifs here right? Never gonna get more than one code?
Honestly, if so, might be best to replace this all with a switch if you can.

Reply:
Ahh okay, good.
And yeah, generally I think switch is a slightly nicer tool with these long if-else chains, but it's really not a big deal to keep as if-else. In terms of the default, the solution is to have it do an RTEAbort cause it covers anyone putting in some junk value.

But yeah, if you'd prefer keeping it as if-else, that's okay :)